src/components/Layout.tsx

```diff
@@ -73,25 +73,12 @@ const Layout: FC<PropsWithChildren> = ({ children }) => {
                     >
                       Compounders
                     </Menu.Item>
-                    {DISABLE_CONCENTRATORS ? (
-                      <Menu.Item
-                        as="span"
-                        className="flex cursor-not-allowed items-center px-3 py-2.5 ui-active:bg-white ui-active:text-pink-900"
-                        disabled
-                      >
-                        <span className="opacity-50">Concentrators</span>
-                        <span className="ml-1 grow-0 whitespace-nowrap rounded bg-pink-200/20 px-1 py-0.5 text-[9px] uppercase leading-tight text-pink-100/80">
-                          Coming soon
-                        </span>
-                      </Menu.Item>
-                    ) : (
-                      <Menu.Item
-                        as={DropdownMenuItemLink}
-                        href={appLink("/yield/concentrators")}
-                      >
-                        Concentrators
-                      </Menu.Item>
-                    )}
+                    <Menu.Item
```

Comment:
I feel like we're gonna wanna test this in staging for a bit before going to prod with it. Should we just keep this env var for now and remove at a later date?